app/views/new_project_wizard/_sidebar.html.erb

```diff
@@ -2,52 +2,52 @@
 <div id="new-project-wizard-sidebar">
     <div class="sidebar-container">
         <div data-state="Expanded" class="expanded-state">
-            <%= render partial: '/new_project_wizard/sidebar_multi_step', locals: { indicator_class: controller.controller_name.start_with?('project_information') ? '-current' : '-completed', 
+            <%= render partial: '/new_project_wizard/sidebar_multi_step', locals: { indicator_class: @request_presenter.sidebar_progress(controller, 1),
```

Comment:
This is the main change...we use the logic in the RequestPresenter to decided what CSS class should be used to display the Wizard steps in the right color (complete, incomplete, current)